README.md

```diff
@@ -13,19 +13,15 @@ Search engine for agencies' published content
 
 Use [rvm](https://rvm.io/) to install the version of Ruby specified in `.ruby-version`.
 
-* [Elasticsearch 6.8](https://www.elastic.co/elasticsearch/)
-* Elasticsearch Plugins:
-    * [analysis-kuromoji](https://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-kuromoji.html)
-    * [analysis-icu](https://www.elastic.co/guide/en/elasticsearch/plugins/master/analysis-icu-analyzer.html)
-    * [analysis-smartcn](https://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-smartcn.html)
+### Docker Services
 
-We recommend using [Docker](https://www.docker.com/get-started) to install and run Elasticsearch:
+The required services (MySQL, Elasticsearch, etc.) can be run using Docker. Please refer to [searchgov-services](https://github.com/GSA/search-services) for detailed instructions on centralized configuration for services.
 
-```
-$ docker-compose up elasticsearch
-```
+The Elasticsearch services provided by `searchgov-services` is configured to run on the default port, [9200](http://localhost:9200/). To use a different host (with or without port) or set of hosts, set the `ES_HOSTS` environment variable. For example, use following command to run the specs using Elasticsearch running on `localhost:9207`:
```

Comment:
When would someone want to override the default port? Are there scenarios other than testing with Elasticsearch 7 running on `9207`?